src/lib.rs

```diff
@@ -22,12 +31,49 @@
     trivial_numeric_casts,
     unreachable_pub,
     unused,
-    variant_size_differences
+    variant_size_differences,
+
+    clippy::pedantic,
+    clippy::nursery,
+    clippy::suspicious,
+    clippy::complexity,
+    //clippy::cargo,
+
+    // Lints from clippy::restriction
+    clippy::string_add, // Don't use + operator to merge strings
```

Comment:
I don't think it's super helpful cluttering up each clippy line with an explanation of itself. Users can go to the clippy website if they need more clarity on a given lint which is both going to be more detailed and up to date.